packages/core/tsup.inject-standalone.config.ts

```diff
@@ -41,36 +48,83 @@ export default defineConfig({
     '@mswjs/interceptors/fetch',
   ],
 
-  dts: true, // Generate .d.ts files
+  // dts: true, // Generate .d.ts files
   // minify: true, // Minify output
-  sourcemap: true, // Generate sourcemaps
-  treeshake: true, // Remove unused code
+  // sourcemap: true, // Generate sourcemaps
+  // treeshake: true, // Remove unused code
 
   // clean: true, // Clean output directory before building
   outDir,
 
   format: [ // Output format(s)
+    'cjs',
     'esm',
-    'cjs'
   ], 
 
   splitting: false,
   keepNames: true, // stops build from prefixing our class names with `_` in some cases
 
   onSuccess: async () => {
-    const filePath = `${__dirname}/${outDir}/injector.js`;
-    const injectorSrc = await fs.promises.readFile(filePath, 'utf8');
-
-    // this fixes a weird annoying issue where the current version of deno used by netlify edge functions
-    // does not have `globalAgent` exported directly from node http/https, while it is part of the default export
-    // so we just swap how it is imported and everything seems to work fine
-
-    const fixedSrc = injectorSrc.replace(`import http, { ClientRequest, IncomingMessage, STATUS_CODES, Agent, globalAgent as globalAgent$1 } from 'node:http';
-import https, { Agent as Agent$1, globalAgent } from 'node:https';`,
-      `import http, { ClientRequest, IncomingMessage, STATUS_CODES, Agent } from 'node:http';
-const globalAgent$1 = http.globalAgent;
-import https, { Agent as Agent$1 } from 'node:https';
-const globalAgent = https.globalAgent;`);
-    await fs.promises.writeFile(filePath, fixedSrc);
-  }
+    // for the edge CJS build
+    // the node imports of zlib and http dont get fully tree-shaken away, so we'll just remove the lingering dead code
+    if (process.env.DMNO_EDGE_COMPAT) {
```

Comment:
I'd love to get rid of these additional adjustments, but it seems the other options I get with esbuild/tsup cant quite do it.